Sources/Positionless/Algorithms/SinglePass.swift

```diff
@@ -23,6 +23,13 @@ extension Collection {
     }
   }
 
+  /// Returns true iff all elements satisfy the given predicate.
+  ///
+  /// - Complexity: No more than `count` application of `predicate`.
+  func all(satisfy predicate: (Element) -> Bool) -> Bool {
+    return !forEach(until: { !predicate($0) })
```

Comment:
Pretty sure `forEach(until: ... )` should be `contains(elementWhere: ...)`.  The only reason to have a `forEach` algorithm would be for mutation or side-effects.